llvm/lib/IR/AttributeImpl.h

```diff
@@ -124,6 +133,17 @@ class AttributeImpl : public FoldingSetNode {
     CR.getLower().Profile(ID);
     CR.getUpper().Profile(ID);
   }
+
+  static void Profile(FoldingSetNodeID &ID, Attribute::AttrKind Kind,
+                      ArrayRef<ConstantRange> Val) {
+    ID.AddInteger(Kind);
+    ID.AddInteger(Val.size());
+    ID.AddInteger(Val[0].getBitWidth());
```

Comment:
This is not necessary, because APInt::Profile already includes it.

Reply:
Ah, done!